eox_nelp/signals/receivers.py

```diff
@@ -266,3 +273,31 @@ def update_async_tracker_context(sender, *args, **kwargs):  # pylint: disable=un
     tracker_context = request.get("kwargs", {}).pop("tracker_context", {})
     current_tracker = tracker.get_tracker()
     current_tracker.enter_context("asynchronous_context", tracker_context)
+
+
+def emit_subsection_attempt_event(usage_id, user_id, *args, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Hee i think this docstring is wrong seems, are ids and not objects.

https://github.com/openedx/edx-platform/blob/4b7ef2697e84bae269983dfce63f1af60bb247df/lms/djangoapps/grades/signals/signals.py#L64-L65

Reply:
By the way then the platform uses IDs in its tests xD 
https://github.com/openedx/edx-platform/blob/4b7ef2697e84bae269983dfce63f1af60bb247df/lms/djangoapps/grades/tests/test_signals.py#L82

Reply:
the first link that you mention belongs to another signal